datumaro/components/dataset.py

```diff
@@ -952,17 +1031,59 @@ def import_from(cls, path: str, format: Optional[str] = None, *,
         else:
             raise UnknownFormatError(format)
 
-        extractors = []
-        for src_conf in detected_sources:
-            if not isinstance(src_conf, Source):
-                src_conf = Source(src_conf)
-            extractors.append(env.make_extractor(
-                src_conf.format, src_conf.url, **src_conf.options
-            ))
+        # TODO: probably, should not be available in lazy mode, because it
```

Comment:
Why is this a TODO? It seems to already be done.

Reply:
The idea is that laziness is not an input parameter in the current implementation, and the extractors are not aligned on this topic. Ideally, it should be a function parameter and should also be aligned with the `eager_mode` CM.